app/controllers/application_controller.rb

```diff
@@ -308,9 +309,15 @@ def current_email
     current_user&.email
   end
 
-  def set_locale
-    if ENV.fetch('LOCALIZATION_ENABLED', 'false') == 'true'
-      I18n.locale = http_accept_language.compatible_language_from(I18n.available_locales)
+  def switch_locale(&action)
+    locale = nil
+    if cookies[:locale]
+      locale = cookies[:locale]
+    elsif ENV['LOCALIZATION_ENABLED'] == 'enabled'
```

Comment:
Effectivement la forme initiale `ENV.fetch('LOCALIZATION_ENABLED', 'false') == 'true'` était incorrecte (habituellement on utilise plutôt `'enabled'`) – mais si on change la valeur ici, il va falloir la changer aussi sur les environnements de production, et c'est fastidieux.

Mais de toute façon on va bientôt supprimer complètement cette variable d'environnement, et la remplacer par une détection de langue du navigateur.

Du coup je suis d'avis de la laisser telle quelle, et de se dire qu'elle sera supprimée dans une prochaine PR.

```suggestion
    elsif ENV.fetch('LOCALIZATION_ENABLED', 'false') == 'true'
```